location-services/read-data.qmd

```diff
@@ -4,48 +4,59 @@ subtitle: "Learn how to read data from ArcGIS Online or Enterprise into R"
 freeze: true
 ---
 
-ArcGIS Online and Enterprise hosted Feature Layers can easily be read into R as [`{sf}`](https://r-spatial.github.io/sf/) objects using`{arcgislayers}`. 
+ArcGIS Online and Enterprise web services can easily be read into R using`{arcgislayers}`. Supported service types include:
 
-This tutorial will teach you the basics of reading data using `arcgis`.
+ - [FeatureServer](https://developers.arcgis.com/rest/services-reference/enterprise/feature-service.htm)
+    - [FeatureLayer](https://developers.arcgis.com/rest/services-reference/enterprise/feature-layer.htm)
+    - [Table](https://developers.arcgis.com/rest/services-reference/enterprise/feature-layer.htm)
+ - [MapServer](https://developers.arcgis.com/rest/services-reference/enterprise/map-service.htm)
+    - [GroupLayer](https://developers.arcgis.com/web-map-specification/objects/groupLayer/)
+ - [ImageServer](https://developers.arcgis.com/rest/services-reference/enterprise/image-service.htm)
+
+
+Metadata for all of the above service types can be accessed using `arc_open()`. Feature data can be read in using `arc_select()` for FeatureLayer, Table, and ImageServer.
+
+This tutorial will teach you the basics of reading data from hosted Feature Layers into R as [`{sf}`](https://r-spatial.github.io/sf/) objects using`{arcgislayers}`. 
 
 ## Objective
 
-The objective of this tutorial is to teach you how to 
+The objective of this tutorial is to teach you how to:
 
-- read in a population dataset from ArcGIS Online
-- apply a filter to a Feature Layer
-- read only specified columns
-- find a Feature Layer url 
+- find a Feature Layer url from ArcGIS Online
+- read in the data from the Feature Layer
+- filter the Feature Layer data by column
+- filter the Feature Layer data by row
```

Comment:
I think these need to be rephrased. `filter` is a row-wise operation where we select rows based on some logical predicates and `select` is a column-wise operation where we fetch specific columns based on named 
```suggestion
- select the Feature Layer data by column
- filter the Feature Layer data by attributes
```

Reply:
Thank you for the fix! the phrasing I had was not sitting super well with me either.